README.md

```diff
@@ -31,13 +34,16 @@ out:
   apikey: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXX'
   list_id: 'XXXXXXXXXX'
   update_existing: false
+  double_optin: false
+  email_column: e mail
+  fname_column: first name
+  lname_column: last name
```

Comment:
@ylzen in the config i test, no need to add ''. But i think better i should revert the old config for column's names